Docs/medium guides (#206)
fetchai/fetchd: docs/docs/convert-native-fet.md

# How to Convert Native Fet to Erc20 Fet Using Fetch.ai Browser Extension Wallet

> Using the Fetch.ai token bridge, here is an easy process explaining how to send tokens across the native to ERC20 side using the [Fetch browser wallet](https://bit.ly/3G31QII).

Reviewer: ```suggestion
This is an easy step-by-step guide explaining how to convert your native FET tokens to ERC20 FET tokens using the [Fetch browser wallet](https://bit.ly/3G31QII).
```

1. Create an account on the Fetch browser wallet if you do not have one. 
   Save your 12/24 word mnemonic safely somewhere. 
    **Do not share this with anyone and if you lose it, you cannot recover your assets.**
     
     <img src="../images/keplr_wallet.png" alt="Keplr Wallet" class="center" style="display: block; margin-left: auto; margin-right: auto;width:400px;">
     
2. Send the amount of tokens you wish to transfer from your cosmostation mobile wallet or your ledger to the generated address on your fetch browser wallet.
3. Once you have done so, browse to [https://token-bridge.fetch.ai/](https://token-bridge.fetch.ai/)
4. Switch to the **Native to ERC20** tab.

> 5. Click on **connect browser wallet **and permit the token bridge to connect with your browser wallet.

Reviewer: ```suggestion
5. Click on **connect browser wallet **and allow the token bridge to connect to your browser wallet.
```

   
    <img src="../images/connect_to_keplr.png" alt="Requesting connection" class="center" style="display: block; margin-left: auto; margin-right: auto;width:400px;">
    
6. The Token bridge will be automatically populated with your currently selected metamask address on ERC20 and your fetch wallet address on native. 
    **Make sure those are correct.**
     
     <img src="../images/token_bridge.png" alt="Token bridge" class="center" style="display: block; margin-left: auto; margin-right: auto;width:400px;">
     
7. Enter the amount.
8. Click **Transfer**.

> **Disclaimer for the ledger users: The wallet integration for ledger nano is currently being worked upon. If you'd like to transfer directly from your ledger, please use the "Connect ledger" button from the Bridge UI instead of using the "Import ledger" button of the wallet.**

Reviewer: ```suggestion
**Disclaimer for the ledger users: The wallet integration for ledger nano is currently being worked on. If you'd like to transfer directly from your ledger device, please use the "Connect ledger" button from the Bridge UI instead of using the "Import ledger" button of the wallet.**
```
